source/eitri/ed_handler.py

```diff
@@ -30,7 +30,6 @@
 import glob
 import os
 from navitiacommon import utils, launch_exec
```

Comment:
from utils, only 1 function used, from launch_exec, only one function used. Not sure about performance (I'll try to do some tests or find doc about it), but I'm not a fan of 'launch_exec.launch_exec'
```suggestion
from navitiacommon.utils import type_of_data 
from navitiacommon.launch_exec import launch_exec
```